products/BellHybrid/services/audio/ServiceAudio.cpp

```diff
@@ -284,6 +287,7 @@ namespace service
             activeInput && activeInput.value()->audio->GetCurrentOperationState() == audio::Operation::State::Paused) {
             retCode = activeInput.value()->audio->Resume();
         }
+        manageCpuSentinel();
```

Comment:
tbh - cant this be somehow done other way round? `if bussy -> manage sentinel` not `manage sentinel -> if bussy` ? :)

Reply:
The status is taken unfortunately from the lowest layers of the audio stack and the callback would need to be installed there.